services/static-webserver/client/Manifest.json

```diff
@@ -41,7 +41,7 @@
   "requires": {
     "@qooxdoo/compiler": "^1.0.0-beta",
     "@qooxdoo/framework": "^6.0.0-beta",
-    "ITISFoundation/qx-iconfont-fontawesome5": "^1.0.0",
+    "ITISFoundation/qx-iconfont-fontawesome5": "^0.2.2",
```

Comment:
are you downgrading?

Reply:
Yep, same icons catalog though. Version 1.0.0 uses qx-7 while we are still on 6, this was throwing a warning message and that's why I downgraded it 👍.